console_windows.go

```diff
@@ -154,6 +154,10 @@ func (m *master) Fd() uintptr {
 	return m.in
 }
 
+func (m *master) Name() string {
+	return ""
+}
```

Comment:
not particularly sure about this. Now that i think a bit more, maybe exposing `File() *os.File` is better for cross-platform. I think in go its always "/dev/stdin".